cfgov/jinja2/v1/events/_macros.html

```diff
@@ -155,10 +155,13 @@ <h2 class="event-venue_heading">
             {# TODO: Replace this call to the Google Maps Static API
                (which now requires an API key we don't have)
                with Mapbox (for which we do have an API key).
+               Is this macro even used?? post-preview.html seems
+                to be instead.
             {% else %}
-              <img src="{{ event.location_image_url(size='640x320') }}"
-                   alt="Google Maps image of
-                        {{ event.venue_city }}{{ ', ' ~ event.venue_state if event.venue_state else '' }}">
+              <img data-js-hook="o-post-preview_image-event-map"
```

Comment:
Not sure of the use case here but I thought it could be helpful to point out our existing data-js-hook resources if they would tie into the architecture here. The original idea behind data-js-hook was that it would contain values that had special meaning to JS. Similar to CSS classes, the values are space separated, but they are not CSS classes and only have meaning to the JS associated with a particular component. See example [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/js/modules/util/standard-type.js#L19-L24). There is also a [data-hook utility](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/unprocessed/js/modules/util/data-hook.js), which can be used to add/remove/query the values, if needed.